lasagne/layers/pool.py

```diff
@@ -14,6 +14,7 @@
     "FeaturePoolLayer",
     "FeatureWTALayer",
     "GlobalPoolLayer",
+    "SpatialPyramidPoolingLayer"
```

Comment:
Please add a `,` in the end. It's optional, but it allows to add lines without touching the ones that are there (just like you could add `"SpatialPyramidPoolingLayer"` without having to touch the `"GlobalPoolLayer",` line).